src/Microsoft.Data.Analysis/DataFrameColumn.cs

```diff
@@ -247,6 +247,21 @@ public virtual DataFrameColumn Sort(bool ascending = true)
         /// </param>
         protected internal virtual void AddDataViewColumn(DataViewSchema.Builder builder) => throw new NotImplementedException();
 
+        /// <summary>
+        /// Appends a value to this <see cref="DataFrameColumn"/> using <paramref name="cursor"/>
+        /// </summary>
+        /// <param name="cursor">The row cursor which has the current position</param>
+        /// <param name="schemaColumn">The <see cref="DataViewSchema.Column"/> in <see cref="DataViewSchema"/></param>
+        /// <param name="ValueGetter">The cached ValueGetter for this column.</param>
+        internal virtual void AddValueUsingCursor(DataViewRowCursor cursor, DataViewSchema.Column schemaColumn, Delegate ValueGetter) => throw new NotImplementedException();
```

Comment:
`internal` means no one can ever override this with their own column type, right?

Reply:
Yup. Fixed it as I was reviewing it internally